docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+title: Mobile App
+weight: 300
+---
+
+# Grafana OnCall Mobile App
+
+Mobile app is one of the ways to receive notifications from Grafana OnCall. OnCall provides multiple ways of delivering notifications to give our users an option to use some of them as a backup. We suggest backing up your Slack, MS Teams, phone calls with notifications delivered to the mobile app or vise versa.
+
+The current version of the app serves the only one purpose to act as a notification method. I's missing some features like a calendar editor or even a list of alert groups. This is a known limitation. Stay tuned, we're working on it!
+
+Download Grafana OnCall Mobile app in Google Play and Apple App Store (links todo after publishing). 
```

Comment:
https://play.google.com/apps/testing/com.grafana.oncall.prod
this is gonna be the android one

Reply:
Do we have the Apple App store link yet?

Reply:
nope